jenkins-jobs/nightly-job-run.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Schedule 0 0 * * *
+# Runs on the gen3-qa-worker node
+# Requires Jenkins secrets
+#   GITHUB_USERNAME
+#   GITHUB_TOKEN
+
+#if [[ ! -d cdis-manifest ]]; then
+#  git clone https://github.com/uc-cdis/cdis-manifest.git
+#fi
+#cd ./cdis-manifest
+# List of commons to select from
```

Comment:
Maybe we could remove the commented code, and move `# List of commons to select from` to just above `commons=("gen3.theanvil.io" ...)`